products/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db  import models
+from utils      import TimeStampModel
+
+class Category(models.Model):
+    name = models.CharField(max_length = 120)
+
+    class Meta:
+        db_table = "categories"
+
+class CategoryProduct(models.Model):
+    category      = models.ForeignKey("Category", on_delete = models.CASCADE)
+    product_group = models.ForeignKey("ProductGroup", on_delete = models.CASCADE)
+
+    class Meta:
+        db_table = "category_products"
+
+class ProductGroup(TimeStampModel):
+    name               = models.CharField(max_length = 120)
+    sales_unit         = models.CharField(max_length = 120)
+    thumbnail          = models.URLField(max_length = 500)
+    delivery_type      = models.CharField(max_length = 120, default = "샛별배송/택배배송")
+    information        = models.TextField()
+    main_description   = models.TextField()
+    detail_description = models.TextField()
+    discount_rate      = models.PositiveIntegerField()
+
+    class Meta:
+        db_table = "product_groups"
+
+class Product(models.Model):
+    product_group    = models.ForeignKey("ProductGroup", on_delete = models.CASCADE)
+    name             = models.CharField(max_length = 120)
+    point            = models.CharField(max_length = 200)
+    price            = models.PositiveIntegerField()
+    ingredient       = models.CharField(max_length = 120)
+    sold_out         = models.BooleanField(default = False)
+
+    class Meta:
+        db_table = "products"
+
+class ProductGroupImage(models.Model):
+    product_group    = models.ForeignKey("ProductGroup", on_delete = models.CASCADE)
+    image_url        = models.URLField(max_length = 500)
+    
+    class Meta:
+        db_table = "product_group_images"
```

Comment:
`queryset`을 다룰 때, 코드의 가독성을 위해서 아래와 같이 수정해주세요!
`product_group_image.image_url` vs `product_group_image.url`
```suggestion
class ProductGroupImage(models.Model):
    product_group    = models.ForeignKey("ProductGroup", on_delete = models.CASCADE)
    url        = models.URLField(max_length = 500)
    
    class Meta:
        db_table = "product_group_images"
```

Reply:
넵!